extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -133,6 +133,24 @@ protected function requestParsoid( $method, $title, $params ) {
 		return $req->getContent();
 	}
 
+
+	protected function getPreloadHTML( $params ) {
+		$preloadTitle = Title::newFromText( urldecode( $params['preload'] ) );
```

Comment:
In the url it looks like preload=MediaWiki:Character-preload

When I process it PHP side $params['preload'] is MediaWiki%3ACharacter-preload

Title::newFromText expects the decoded name.

I don't do any other processing of $params['preload'], so it should be handled the same way as all other elements in $params.

Is colon (:) a valid part of a plain MediaWiki page, or is it only used for "special" pages such as the MediaWikia: page here?